src/test/scala/com/fulcrumgenomics/bam/api/SamRecordTest.scala

```diff
@@ -106,12 +107,34 @@ class SamRecordTest extends UnitSpec {
     rec.asSam.getCigar shouldBe       null
   }
 
-  "SamRecord.apply" should "not return values for attributes that exist and not for those that don't exist" in {
+  it should "return values and information about SAM tags" in {
```

Comment:
Switching to `it` here is going to list test under `"SamRecord.cigar"` :/  How about:

```scala
"SamRecord" should "provide easy and safe access to extended attributes" in {
```